### ISSUE TITLE
CPS: Incorrect code generated for checking for growable lists

### ISSUE THREAD

**Issue by kmillikin** [dart-lang/sdk#23708]
The SSA backend has special code for some type checks.  For example, in pkg/compiler/lib/src/ssa/codegen.dart there is:

```
  void checkExtendableArray(HInstruction input) {
    checkFieldDoesNotExist(input, 'fixed\$length');
  }
```

But the CPS backend just compiles a normal type check, which doesn't do the right thing.

@asgerf @karlklose @sigmundch

**Comment by kmillikin** [comment]
@rakudrama

**Comment by asgerf** [comment]
https://codereview.chromium.org/1206873003/

**Comment by karlklose** [comment]
Closed in fceb7f00dc7a0d43385589a47146b63a20b447a9.
